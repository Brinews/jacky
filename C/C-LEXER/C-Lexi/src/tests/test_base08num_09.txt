
project1x08

   0x08>12x08
