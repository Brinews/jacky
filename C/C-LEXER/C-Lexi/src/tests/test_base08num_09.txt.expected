{project1x08 , ID , 2}
{0x08 , BASE08NUM , 4}
{ , GREATER , 4}
{12x08 , BASE08NUM , 4}
{ , END_OF_FILE , 6}
